moduleframework/helpfile_linter.py

```diff
@@ -49,7 +49,10 @@ def get_maintainer_name(self, name):
 
     def get_tag(self, name):
         name = '# %s' % name
+        tag_found = True
         if not self.help_md:
+            common.print_info("help md does not exist.")
             return False
-        tag_exists = [x for x in self.help_md if name.upper() in x]
-        return tag_exists
+        if not [x for x in self.help_md if name.upper() in x]:
```

Comment:
I'm not sure with this statement, what you have there means?
*  `if name.upper() not in self.help_md`
* or it means that you are searching for substring, in that case you can use something like: `if name.upper() not in " ".join(self.help_md)`, in case you are not searching for substring this and original code will lead to mistake, when for example name.upper() == "SOME" and x will contain for example "SOMEONE" 
* or it means that you are searching by `if name.upper() in x` in more complex object? like dict, in this case this usage is okay.

Reply:
It means, go through help_md file line by line. Search for given string in line somewhere and before searching it convert to upper.

Reply:
thanks for explanation. So it make sense to have it in this form and equivalent is something like: `if name.upper() not in "\n".join(self.help_md)`